cdc/log.cc

```diff
@@ -63,6 +63,27 @@ using namespace std::chrono_literals;
 
 logging::logger cdc_log("cdc");
 
+sstring cdc::operation_to_text(cdc::operation op) {
+    switch (op) {
```

Comment:
const sstring& / const std::string&? Latter could use string literals.

Reply:
former constants declared before.

Reply:
Done.

Reply:
or simply std::string_view by value. Guaranteed least code and least overhead (unless gcc is completely useless).

Reply:
Changed to string_view.